content/en/docs/v3/about/maturity-matrix.md

```diff
@@ -5,6 +5,28 @@ description: Jenkins X 3.x features maturity assessment on different cloud provi
 weight: 170
 ---
 
+<style>
```

Comment:
Adding script tags to markdown is not a good approach, so let's not do this. The hugo way of doing this would be shortcodes.